### ISSUE TITLE
Version 0.0.21 has no sdist on pypi

### ISSUE THREAD

**Issue by pbsds** [freddyaboulton/gradio-pdf#35]
https://pypi.org/project/gradio-pdf/0.0.21/#files

**Comment by freddyaboulton** [comment]
Yes I purposely don't upload it because it's complicated to build from source. Pip would fail. Why do you need the source on pypi?

**Comment by pbsds** [comment]
The purpose is distribution. For source provenance reasons we prefer to build from source if possible. I you prefer to not publish the sdist, could you consider starting to tag releases here on github?

**Comment by freddyaboulton** [comment]
Yes I can do that!
